symphony-ai/src/main/java/org/symphonyoss/symphony/bots/ai/impl/AiCommandInterpreterImpl.java

```diff
@@ -37,7 +35,8 @@ public boolean isCommand(AiCommand aiCommand, AiMessage command, String commandP
 
     ArgumentType[] argumentTypes = aiCommand.getArgumentTypes();
 
-    if ((argumentTypes != null) && (commandAndArgs.length < aiCommand.getArgumentTypes().length + 1)) {
+    if ((argumentTypes != null) && (commandAndArgs.length
+        < aiCommand.getArgumentTypes().length + 1)) {
```

Comment:
Isn't better to use <= something instead of < something + 1?

Reply:
Done